src/Umbraco.Cms.Integrations.Crm.Hubspot/App_Plugins/UmbracoCms.Integrations/Crm/Hubspot/Render/HubspotForm.cshtml

```diff
@@ -1,31 +1,14 @@
-﻿@using System.Configuration
-@inherits Umbraco.Web.Mvc.UmbracoViewPage<Umbraco.Cms.Integrations.Crm.Hubspot.Models.ViewModels.HubspotFormViewModel>
+﻿@inherits Umbraco.Web.Mvc.UmbracoViewPage<Umbraco.Cms.Integrations.Crm.Hubspot.Models.ViewModels.HubspotFormViewModel>
 
 @{
-    var region = ConfigurationManager.AppSettings["Umbraco.Cms.Integrations.Crm.Hubspot.Region"];
-
-    var hbsptScriptPath = string.IsNullOrEmpty(region) ? "//js.hsforms.net/forms/shell.js" : "//js-eu1.hsforms.net/forms/shell.js";
+    var hbsptScriptPath = string.IsNullOrEmpty(Model.Region) ? "//js.hsforms.net/forms/shell.js" : "//js-eu1.hsforms.net/forms/shell.js";
 }
 
 <script charset="utf-8" type="text/javascript" src="@hbsptScriptPath"></script>
-@if (string.IsNullOrEmpty(region))
-{
-    <script>
-        hbspt.forms.create({
-            portalId: "@Model.PortalId",
-            formId: "@Model.Id"
-        });
-    </script>
-
-}
-else
-{
-    <script>
-        hbspt.forms.create({
-            region: "@region",
-            portalId: "@Model.PortalId",
-            formId: "@Model.Id"
-        });
-    </script>
-
-}
\ No newline at end of file
+<script>
+    hbspt.forms.create({
+        region: "@Model.Region",
```

Comment:
Looks like you are now applying the `region` setting irrespective of if we have a value now.  If that works as expected then it's fine, just noting in case you did this inadvertently.